urls.py

```diff
@@ -55,6 +55,7 @@
     (r'^data/', include('corehq.apps.data_interfaces.urls')),
     (r'^', include(hqwebapp_domain_specific)),
     (r'^case/', include('corehq.apps.hqcase.urls')),
+    (r'^case/', include('corehq.apps.case_search.urls')),
```

Comment:
nit: is there any reason not to give this app it's own base URL? just seems like there's more of a chance for mistakes if pattern matching is happening across two files.